process/smartContract/process.go

```diff
@@ -1770,7 +1775,7 @@ func (sc *scProcessor) createSmartContractResults(
 
 	if bytes.Equal(outAcc.Address, vm.StakingSCAddress) {
 		storageUpdates := process.GetSortedStorageUpdates(outAcc)
-		result := createBaseSCR(outAcc, tx, txHash)
+		result := createBaseSCR(outAcc, tx, txHash, outAcc.Nonce)
```

Comment:
This is not backward compatible.

Reply:
The resulting scr will be outacc.nonce * 2

Reply:
Shouldn't the last parameter be 0 instead of `outAcc.Nonce`?

Reply:
Right, remained from the first implementation. Thanks! :+1:

Reply:
fixed